package.json

```diff
@@ -10,7 +10,7 @@
   "scripts": {
     "prebench": "npm run build:cjs",
     "bench": "node dist/cjs/bench/index.js",
-    "build": "npm run build:cjs && npm run build:esm && npm run build:umd && npm run build:types",
+    "build": "npm run build:cjs && npm run build:esm && npm run build:umd && npm run build:types && npm run package",
```

Comment:
Do we need to run `npm run package` here? That should only be for publishing

Reply:
I can remove, I had it in there because I consumed the .tar.gz in xmtp-inbox-web